waspc/examples/todoApp/src/rpcTests/operations/fixtures.ts

```diff
@@ -10,5 +10,6 @@ export const SERIALIZABLE_OBJECTS_FIXTURE = {
   negInf: -Infinity,
   // we ensure that it'd be too big to be represented as a `number`
   decimal: new Prisma.Decimal(Number.MAX_SAFE_INTEGER).mul(2),
+  // @ts-ignore Server has a tsconfig target of es2017, so BigInt isn't available
```

Comment:
Can we update the server tsconfig? We can do it in a different PR and remove this comment.

If it turns out to cause problems, we can create an issue and link it here.